### PR TITLE
Fix for kekuleAtrop wedge error

### PR DESCRIPTION
**Pull request by tadhurst-cdd**
#### Reference Issue
Some atropisomers with aromatic rings return invalid smiles when the kekule option is selected.  For example:
```CC1C(C2C(Cl)=CC=CC=2C)=C(Cl)C=CC=1 |wD:3.3|```
kekuleWedgeError
 returns 
```CC1=C(C2=C(C)C=CC=C2Cl)C(Cl)=CC=C1 |wD:2.1|```
kekuleWedgeErrorBadOut

This output mol is not valid - it has a wedge on a double bond.  Some software packages will drop the wedge because it is on a double bond (rdkit), and others will change the bond to single as in the picture (MarvinJS).

This happens because the kekule operation occurs on the fragments of the molecule but not on the original combined molecule.  The original combined mol is then used to determine the location of the wedge/hash bonds, and it places the wedge on an aromatic bond, which is allowed.  However, the corresponding bond in the fragment has been changed to a double bond,  which is not valid for the wedge.

#### What does this implement/fix? Explain your changes.
Now when a CXSmiles is requested with the kekule option, the kekule action is taken on the combined molecule BEFORE it is fragmented, so that the original combined molecule and the fragments are all in kekule form.

### CODE REVIEW

File: Code/GraphMol/SmilesParse/CXSmilesOps.cpp [hunk 1]
```diff
@@ -2490,7 +2500,7 @@ std::string getCXExtensions(const ROMol &mol, std::uint32_t flags) {
   }
 
   const Conformer *conf = nullptr;
-  if (mol.getNumConformers()) {
+  if (mol.getNumConformers() && (flags & SmilesWrite::CX_COORDS)) {
```
Comment (greglandrum) [inline Code/GraphMol/SmilesParse/CXSmilesOps.cpp:2503]: I think this change means that the code starting at line 2523 below can be simplified to:
```
  else if (flags & SmilesWrite::CXSmilesFields::CX_BOND_ATROPISOMER) {
    Atropisomers::wedgeBondsFromAtropisomers(mol, conf, wedgeBonds);
    const auto cfgblock = get_bond_config_block(
        mol, atomOrder, bondOrder, conf != nullptr, wedgeBonds, true);
    appendToCXExtension(cfgblock, res);
  }
```

**Review by greglandrum — CHANGES_REQUESTED** [review]
One small suggestion

File: Code/GraphMol/SmilesParse/CXSmilesOps.cpp (hunk 1 above)
Comment (tadhurst-cdd) [reply Code/GraphMol/SmilesParse/CXSmilesOps.cpp:2503]: that works

**Review by greglandrum — APPROVED** [review]
LGTM

**Pull request merged**